handler.go

```diff
@@ -145,6 +145,16 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, u *User) {
 		return
 	}
 
+	// If authentication is enabled and there are no users yet, make sure
+	// the first statement is creating a new cluster admin.
+	if h.AuthenticationEnabled && h.server.UsersLen() == 0 {
+		stmt, ok := query.Statements[0].(*influxql.CreateUserStatement)
+		if !ok || stmt.Privilege == nil || *stmt.Privilege != influxql.AllPrivileges {
+			h.error(w, "must create cluster admin", http.StatusBadRequest)
```

Comment:
Probably so.  In the spec we decided to only return 200, 400, or 500 so I thought we'd discuss it here.  :)  @pauldix @benbjohnson  ?